src/popup/components/ProtocolSpecificView.vue

```diff
@@ -58,7 +63,8 @@ export default defineComponent({
     },
   },
   setup(props) {
-    const { params, meta } = useRoute();
+    const { params, meta }: { params: RouteParams, meta: WalletRouteMeta } = useRoute();
```

Comment:
What do you think about this approach?
```suggestion
    const route = useRoute();
    const routeMeta = route.meta as WalletRouteMeta;
    const routeParams = route.params;
```